includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -819,10 +819,10 @@ public function has_header_logo() {
 	 */
 	public function get_header_logo_id() {
 		if ( ! empty( $this->settings['header_logo'] ) ) {
-			$header_logo_id = $this->get_settings_text( 'header_logo', '', false );
+			$header_logo_id = intval( $this->get_settings_text( 'header_logo', '', false ) );
```

Comment:
@alexmigf I have used `absint` instead, however, I think that other checks are not needed. If `absint` gets an invalid or empty string as an input, it will return 0. And if `$header_logo_id` is 0, we will assign it to false because then, the logo is not set.

What do you think?

Reply:
Seems fine to me.